src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.cs

```diff
@@ -718,8 +718,8 @@ private static void ValidateCspFlags(CspProviderFlags flags)
             if (hr != S_OK)
             {
                 hKey.Dispose();
-                if (IsFlagBitSet((uint)parameters.Flags, (uint)CspProviderFlags.UseExistingKey) ||
-                                                         (uint)hr != (uint)CryptKeyError.NTE_NO_KEY)
+                if (unchecked(IsFlagBitSet((uint)parameters.Flags, (uint)CspProviderFlags.UseExistingKey) ||
+                                                                   (uint)hr != (uint)CryptKeyError.NTE_NO_KEY))
```

Comment:
Is it possible that the differ highlights a tab I removed? I checked the file in VS and in GitHub's raw view and I don't see a tab in that location. Also double-checked my VS settings and the editor is set to 'Insert spaces'. Am I missing something?